WebDriverTests/src/test/java/Task40/ExplicitWaitForSearchByOffice.java

```diff
@@ -0,0 +1,66 @@
+package Task40;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class ExplicitWaitForSearchByOffice {
+    private static final String DESTINATION_URL = "https://192.168.100.26/";
+    private static final String USERNAME = "EugenBorisik";
+    private static final String PASSWORD = "qwerty12345";
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void invokeBrowser() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @AfterMethod
+    public void closeBrowser() {
+        driver.close();
+    }
+
+    @Test(groups = { "smoke", "waits" })
+    public void verifySearchByOfficeDisplayed() {
```

Comment:
We are trying to follow development practices as much as possible, so we use rule of "AAA" - http://wiki.c2.com/?ArrangeActAssert
Your code and performance is very good, so I want you to read information and adjust changes by yourself without my additional comments here (I want you to foresee, what I preffer to change here). Such approach will grant you better uderstanding, how tests should be written.